libs/shared/notifier/src/lib/notifier.service.ts

```diff
@@ -99,3 +100,28 @@ export class NotifierService {
     return map;
   }
 }
+
+export const NotifierServiceWithNestLogger = {
```

Comment:
Most of the existing modules in `libs/*` aren't using `ConfigService` and instead define the necessary configs in a `*.config.ts` module. Was it considered replacing `ConfigService` with `notifier.config` instead?

Reply:
Other Nest apps in `packages` are currently using `NotifierService`; this was the quickest solution to unblock FXA-10177.